draft-ietf-mmusic-t140-usage-data-channel.xml

```diff
@@ -279,32 +279,33 @@
                 'dcsa' attribute. 
               </t>
               <t>
-                If the offerer has marked a data channel as sendrecv (implicit or explicit) or
-                recvonly, it MUST be prepared to receive T.140 data as soon as the state of
-                the T.140 data channel allows it. 
+                If the offerer has marked a data channel as sendrecv (or if the offerer did not
+                explicitly mark the data channel) or recvonly, it MUST be prepared to receive
+                T.140 data as soon as the state of the T.140 data channel allows it. 
               </t>
           </section> 
           <section anchor="sec.sdp.dcsa.dir.neg.ans" title="Generating an Answer">
               <t>
                 When the answerer accepts an offer, and marks the direction of the text 
-                in the corresponding answer, the marking is based on the marking (explicit or implicit) 
-                in the offer.
+                in the corresponding answer, the marking is based on the marking (or the lack 
```

Comment:
Suggest changing "marking is based on the marking" to "direction is based on the marking"

Reply:
Will fix as suggested.